networkx/algorithms/flow/edmondskarp.py

```diff
@@ -8,6 +8,11 @@
 __all__ = ["edmonds_karp"]
 
 
+@nx._dispatch(
+    graphs="R",
+    preserve_edge_attrs={"R": {"capacity": float("inf"), "flow": None}},
```

Comment:
One more @dschult... can we choose a sane default value for `"flow"`? My guess is `0`.

Reply:
This is a hard module to know how to handle because the functions are building the solution by adding attributes to the "residual graph" R.  There is no user-defined attribute "flow". The flow values are the result of the computation. 

Other functions set up the flow initially to be 0 for every edge in R. For the public function there would be no initial value -- or edge attribute holding the flow.   

This specific function should really be a private function (It's not listed in `__all__`).  Still, if we want to put the decorator on it:  we should use a default value of 0.

Reply:
Thanks. Updated to use `0`.